### ISSUE TITLE
vbox additions certificate is expired

### ISSUE THREAD

**Issue by rjhornsby** [joefitzgerald/packer-windows#230]
The included vbox guest additions [certificate](https://github.com/joefitzgerald/packer-windows/blob/master/scripts/oracle-cert.cer) is expired:
```
$ keytool -printcert -file oracle-cert.cer
Owner: CN=Oracle Corporation, OU=VirtualBox, OU=Digital ID Class 3 - Microsoft Software Validation v2, O=Oracle Corporation, L=Redwood Shores, ST=California, C=US
Issuer: CN=VeriSign Class 3 Code Signing 2010 CA, OU=Terms of use at https://www.verisign.com/rpa (c)10, OU=VeriSign Trust Network, O="VeriSign, Inc.", C=US
Serial number: 51ca009816fdbd80f120e015ee75823e
Valid from: Sun Dec 22 18:00:00 CST 2013 until: Thu Dec 22 17:59:59 CST 2016
...
```

The certificate will still install in Windows, but causes the scripted vbox additions installation to fail silently.

Logged [PR 229](https://github.com/joefitzgerald/packer-windows/pull/229) with updated certificate.

**Comment by nap** [comment]
Replace the `virtualbox` goto label definition with the following

```
:virtualbox
move /Y C:\Users\vagrant\VBoxGuestAdditions.iso C:\Windows\Temp
cmd /c ""C:\Program Files\7-Zip\7z.exe" x C:\Windows\Temp\VBoxGuestAdditions.iso -oC:\Windows\Temp\virtualbox"
cmd /c C:\Windows\Temp\virtualbox\cert\VBoxCertUtil add-trusted-publisher C:\Windows\Temp\virtualbox\cert\vbox-sha1.cer --root C:\Windows\Temp\virtualbox\cert\vbox-sha1.cer
cmd /c C:\Windows\Temp\virtualbox\cert\VBoxCertUtil add-trusted-publisher C:\Windows\Temp\virtualbox\cert\vbox-sha256.cer --root C:\Windows\Temp\virtualbox\cert\vbox-sha256.cer
cmd /c C:\Windows\Temp\virtualbox\cert\VBoxCertUtil add-trusted-publisher C:\Windows\Temp\virtualbox\cert\vbox-sha256-r3.cer --root C:\Windows\Temp\virtualbox\cert\vbox-sha256-r3.cer
cmd /c C:\Windows\Temp\virtualbox\VBoxWindowsAdditions.exe /S
```

**Comment by StefanScherer** [comment]
Fixed with #247
